frontend/src/components/GoalCards/SessionObjectiveCard.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './SessionObjectiveCard.css';
+
+function SessionObjectiveCard({
+  objective,
+  objectivesExpanded,
+}) {
+  const {
+    title,
+    trainingReportId,
+    grantNumbers,
+    endDate,
+    topics,
+    sessionName,
+  } = objective;
+
+  const trainingReportUrl = `/training-report/${trainingReportId.substring(trainingReportId.lastIndexOf('-') + 1)}/event-summary`;
+
+  return (
+    <dl className="ttahub-goal-card__objective-list ttahub-goal-card__objective-list--session-objective usa-list usa-list--unstyled padding-2 margin-top-2 border radius-lg" hidden={!objectivesExpanded}>
```

Comment:
I think a ```<dl>``` is probably the right tag to use on the other Objective card as well, but I'm not sure about some of the markup we have nested inside of it being appropriate, so I didn't refactor that one just yet.

Reply:
I guess I could go either way to seems its terms and their definitions for <dl>

The <dl> tag defines a description list. The <dl> tag is used in conjunction with <dt> (defines terms/names) and <dd> (describes each term/name).